sql-server-2014-alwayson-existing-vnet-and-ad/azuredeploy.parameters.json

```diff
@@ -1,69 +1,44 @@
 {
   "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
   "contentVersion": "1.0.0.0",
-  "parameters": {
-    "newStorageAccountNamePrefix": {
-      "value": "GEN-UNIQUE-18"
-    },
 
-    "storageAccountType": {
-      "value": "Premium_LRS"
-    },
+  "parameters": {
 
-    "vmNamePrefix": {
+    "namePrefix": {
       "value": "GEN-UNIQUE-8"
     },
 
-    "sqlVMSize": {
-      "value": "Standard_DS3"
-    },
-
-    "sqlWitnessVMSize": {
-      "value": "Standard_DS1"
-    },
-
-     "existingDomainName": {
+    "existingDomainName": {
       "value": "contoso.com"
     },
 
-     "existingAdPDCVMName": {
-      "value": "vmad01"
+    "adminUsername": {
```

Comment:
remove the params here that don't have values that will pass deployment - leaving them out will provide a prompt in most experiences.